rust2/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[workspace]
+members = ["core", "drivers/*"]
+resolver = "2"
+
+[workspace.package]
+version = "0.1.0"
+description = "Rust implementation of Arrow Database Connectivity (ADBC)"
+edition = "2021"
+authors = ["Apache Arrow <dev@arrow.apache.org>"]
+license = "Apache-2.0"
+
+[workspace.dependencies]
+arrow = { version = "51.0.0", default-features = false, features = ["ffi"] }
```

Comment:
According to my analysis, we have to depend on `arrow` because the following items are not defined in a sub-crate:
- `arrow::ffi_stream::FFI_ArrowArrayStream`
- `arrow::ffi_stream::ArrowArrayStreamReader`
- `arrow::ffi::from_ffi`
- `arrow::ffi::to_ffi`

Maybe we can open a PR in arrow-rs to fix that (I don't know if this is easy or not)

Reply:
Ah I missed that. I'll try to create a PR to introduce an `arrow-ffi` crate. For now we can just keep using `arrow` here then.

Reply:
I've opened an issue upstream to ask for ideas: https://github.com/apache/arrow-rs/issues/5663